packages/slic-tools/dynamo.js

```diff
@@ -0,0 +1,41 @@
+const { DynamoDBClient } = require('@aws-sdk/client-dynamodb')
+const { DynamoDBDocumentClient } = require('@aws-sdk/lib-dynamodb')
+const { captureAWSv3Client } = require('aws-xray-sdk-core')
+
+const defaultOptions = {
+  // Prevent long-running retry loops caused by the default SDK DDB retry count of 10 with
+  // exponential backoff with deals up to 25 seconds
+  maxAttempts: 4
```

Comment:
[maxRetries](https://docs.aws.amazon.com/AWSJavaScriptSDK/latest/AWS/Config.html#maxRetries-property)

v2: The maximum amount of retries to perform for a service request.
v3: Changed to `maxAttempts`. See more in [v3 reference for RetryInputConfig](https://docs.aws.amazon.com/AWSJavaScriptSDK/v3/latest/interfaces/_aws_sdk_middleware_retry.retryinputconfig-2.html#maxattempts). Note that the maxAttempt should be maxRetries + 1.

More info on https://github.com/aws/aws-sdk-js-v3/blob/main/UPGRADING.md#client-constructors